ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15.yaml

```diff
@@ -32,9 +32,19 @@ build_root:
 images:
 - dockerfile_literal: |
     FROM src AS builder
+    RUN mkdir -p /tmp/opmbuild && cd /tmp/opmbuild && pwd && \
+      MIRRORURL=https://mirror2.openshift.com/pub/openshift-v4 && CLIENTURL=${MIRRORURL}/x86_64/clients/ocp/candidate && \
+      curl -s -k -L ${MIRRORURL}/x86_64/clients/ocp/ -o ocp.html && curl -s -k -L ${MIRRORURL}/x86_64/clients/ocp-dev-preview/ -o pre.html && \
+      ecver=$(grep -E "<a href=\"candidate-4\.15" pre.html |cut -d"\"" -f2|cut -d"/" -f1|sort -V|tail -1) && echo "V${ecver}V" && \
```

Comment:
4.15 has been GAed for a long time, perhaps we can remove those EC stuff.

Reply:
@liangxia thanks. updated.